shaders/blur.comp

```diff
@@ -2,15 +2,58 @@
 
 layout(local_size_x = 8, local_size_y = 4, local_size_z = 1) in;
 layout(rgba32f, binding = 0) uniform image2D screen;
+layout(std430, binding=1) buffer agentbufferblock { vec4 agentData[]; };
```

Comment:
Couldn't figure out how to send 2 buffers so I've packed them into a single buffer of vec4s
x/y component is position
z/w component is direction